CHANGELOG.rst

```diff
@@ -31,7 +31,11 @@ Fixed
 * Added support for 32bit UUIDs. Fixes #1314.
 * Fixed typing for ``BaseBleakScanner`` detection callback.
 * Fixed possible crash in ``_stopped_handler()`` in WinRT backend. Fixes #1330.
-
+* Fixed ``KeyError`` being raised during waiting for service
+  discovery in BlueZ backend.
```

Comment:
This looks like two separate changes. Can we split these into two separate commits? This one seems pretty straight forward, but I will need some time to look at the rest of it.

Reply:
split into https://github.com/hbldh/bleak/pull/1377

Reply:
removed change and squashed